ui/public/static/img/traefik_mesh-light.svg

```diff
@@ -1 +1,23 @@
\ No newline at end of file
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" version="1.1" viewBox="0 0 128 128">
```

Comment:
btw, traefik Mesh icon was there in the component_svg folder. Do we need it?

Reply:
Yes, It was being still being in navigator.

Reply:
Okay, it looks like you haven't added it, but it was already there. The cleanup could be done in next PR.